### PR TITLE
Add support for post edit notifications

### PR DESCRIPTION
**Pull request by Tak**
:ballot_box_with_check: Notification channel
:ballot_box_with_check: Preference
:ballot_box_with_check: Filter

image

ref: https://github.com/mastodon/mastodon/pull/16697

### CODE REVIEW

**Comment by connyduck** [comment]
I'm sorry I just merged the other notification type -  can you please resolve the conflicts? 😬

**Comment by Tak** [comment]
:ballot_box_with_check: Conflicts resolved

File: app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java [hunk 1]
```diff
@@ -494,35 +506,25 @@ void setMessage(NotificationViewData.Concrete notificationViewData, LinkListener
...
-                        icon.setColorFilter(ContextCompat.getColor(context,
-                                R.color.tusky_blue), PorterDuff.Mode.SRC_ATOP);
-                    }
-
+                    icon = getIconWithColor(context, R.drawable.ic_home_24dp, R.color.tusky_blue);
                     format = context.getString(R.string.notification_subscription_format);
                     break;
                 }
+                case UPDATE: {
+                    icon = getIconWithColor(context, R.drawable.ic_home_24dp, R.color.tusky_blue);
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java:524]: I have never seen an edit notification in the webinterface, but shouldn't the icon be an edit icon, like a pencil?

Comment (Tak) [reply app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java:524]: Good point, that is what it is in the web ui

**Review by connyduck — APPROVED** [review]

**Pull request merged**